src/main/resources/com/typesafe/sbt/packager/archetypes/java_server/systemloader/systemv/start-debian-template

```diff
@@ -12,6 +12,10 @@
 source /lib/init/vars.sh
 source /lib/lsb/init-functions
 
+# if configuration files exist, source it and use JAVA_OPTS to prepend their contents to $@
+[[ -f /etc/default/${{app_name}} ]] && . /etc/default/${{app_name}}
+# $JAVA_OPTS used in $RUN_CMD wrapper
+export JAVA_OPTS
```

Comment:
it depends how JAVA_OPTS is set in /etc/default/${{app_name}} (if set)
JAVA_OPTS='toto' will overwrite,
JAVA_OPTS="toto $JAVA_OPTS" wouldn't

BTW, JAVA_OPTS seams not set before /etc/default

then I force export because I know JAVA_OPTS is required in wrapper subprocess created by start-stop-daemon (for any other envars required by java process, exporting it in /etc/default/ would be mandatory); if not, JAVA_OPTS would be limited to init.d script process only and not propagated to subprocess.